amundsen_application/static/js/components/common/Card/index.spec.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+
+import Card, { CardProps } from '.';
+
+const setup = (propOverrides?: Partial<CardProps>) => {
+  const props = {
+    title: 'test title',
+    ...propOverrides,
+  };
+  const wrapper = mount<typeof Card>(<Card {...props} />);
```

Comment:
Is there some TypeScript benefit for  `<typeof Card>` vs `<Card>`?

Reply:
Not really sure. In this case if I take out the 'typeof', I get an error: "Card' refers to a value, but is being used as a type here. Did you mean 'typeof Card'?"

I saw that in other cases, we don't actually add this type here at all. 🤷